Initial Review Commit.

jitesh-spryiq opened the pull request:
#### Descriptions:
_Add PR descriptions here_

#### Fix:
_Add fix/changes here_

jitesh-spryiq committed:
Initial Review Commit.

cs-shivaji-kolape commented on security-trails/connector.py:
```diff
@@ -0,0 +1,22 @@
+""" Copyright start
```
Copyright string should be 
"""
Copyright start
MIT License
Copyright (c) 2024 Fortinet Inc
Copyright end
"""

cs-shivaji-kolape commented:
PR descriptions should not be empty. @jitesh-spryiq Please provide detailed descriptions, including:

Connector REST API Doc link of the 3rd party product.
Whether actions were tested during setup or not
Whether unit testing was done or not

jitesh-spryiq committed:
Addressing review comments
1. Copyright string change
2. Connector REST API Doc link .
3. Actions were tested
4. unit testing was done
